sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs

```diff
@@ -277,7 +277,7 @@ Page<DataFeed> NextPageFunc(string nextLink, int? pageSizeHint)
         /// </returns>
```

Comment:
I can't comment on it but the return docstrings for all methods need to be updated

Reply:
Oh, no. Good catch. Updating it.